src/sc2laddercore/LadderManager.cpp

```diff
@@ -137,7 +137,7 @@ ExitCase GameUpdate(sc2::Connection *client, sc2::Server *server, const std::str
 			SC2APIProtocol::Status CurrentStatus;
 			if (!client || !server || client->connection_ == nullptr)
 			{
-				PrintThread{} << botName << " Null server or client returning ClientTimeout" << std::endl;
+				PrintThread{} << *botName << " Null server or client returning ClientTimeout" << std::endl;
```

Comment:
`botName` might be reference. Dunno why it is passed as a pointer.

Reply:
yeh, agree, it would be better to pass this in as a reference.  I'll make the change, thanks :)